lib.py

```diff
@@ -28,21 +28,28 @@
 logger = logging.getLogger(__name__)
 
 
-async def benchmark(ctx: commands.Context, day: int, part: int, code: bytes, ):
+async def benchmark(
+    ctx: commands.Context,
+    day: int,
+    part: int,
+    code: bytes,
+):
     """Run the entire benchmark process, end-to-end."""
     op_name, op_id = ctx.author.name, ctx.author.id
 
     try:
-        db = Database().get()
-        cur = db.cursor()
         results = []
         with tempfile.TemporaryDirectory(suffix=f"-ferris-elf-{op_id}") as tmpdir:
             populate_tmp_dir(tmpdir, code)
             if not await build_code(op_name, op_id, tmpdir):
                 # This reply is not good UX, but it's better than silence.
                 await ctx.reply("Build failed.")
                 return
-            answers_map = load_answers(cur, day, part)
+
+            with Database() as db:
+                # TODO dont hardcode 2023
+                answers_map = db.load_answers(2023, day, part)
```

Comment:
There's already a `year()` function in this file, but OK.

Reply:
That year function is a few days away from reporting 2024, and im assuming we want to be testing on the 2023 data for now

Reply:
I've got a fix queued up for this as part of my work on #41 , but we can flip this over when I open that PR.